lib/partition_setup.pm

```diff
@@ -81,15 +81,17 @@ sub create_new_partition_table {
     send_key_until_needlematch "expert-partitioner-vda", 'right';
 
     # empty disk partitions by creating new partition table
-    my $expert_menu_key = (is_storage_ng) ? 'alt-r' : 'alt-x';    # expert menu keys
+    # in sle15sp1 is called Pa{r}tition Table
+    my $expert_menu_key = (is_storage_ng) ? ((is_storage_ng_newui) ? 'alt-r' : 'alt-e') : 'alt-x';    # expert menu keys
```

Comment:
I think we are getting into some wrong direction here. What would happen when in SP2 they will change ui one more time ? something like this ? 
```suggestion
    my $expert_menu_key = (is_storage_ng) ? ((is_storage_ng_newui) ? ((is_storage_ng_themostnewui ) ? 'alt-r' : 'alt-q' ): 'alt-e') : 'alt-x';    # expert menu keys
```

Reply:
So with this future forecast direction changed how ?

Reply:
Please skip meaningless sentences like direction or words that give fantasy space for interpretation.
Do we have some info from YaST about planned change in storage(_ng) ? If not then I would not add anything that is not there.

Reply:
I am just showing that proposed solution won't scale in future and we need to find some other way to handle this. If you asking for certain suggestion I would go for split everything on version basis without this "newnew" functions which confuse a lot because they "new" only limited amount of time

Reply:
@asmorodskyi I agree with you, however I think this is out of scope of this PR because we already *have* the functional "is_storage_ng_newui". I created https://progress.opensuse.org/issues/45116 to improve that. Can you provide your thoughts there? ok for you?

Reply:
> Please skip meaningless sentences like direction or words that just give space for interpretation.
> Do we have some info from YaST about planned change in storage(_ng) ? If not then I would not add anything that is not there.

sorry to confuse you . just to keep it clear : my "suggestion" was just anti-pattern showing how I do NOT want code look like. I don't have any insights from YaST team but I am pretty sure  that sooner or later they will change this hotkey so "new" will become "old"

Reply:
I did not know what you mean, if you would write it as second comment everything would be fine.